docs/joss_paper/paper.md

```diff
@@ -0,0 +1,120 @@
+
+An encoder-decoder architecture, such as an adapted U-Net and the local-net [@hu2018weakly] is used for dense prediction such as direct DDF or DVF. An encoder-only architecture, such as ResNet and VGG, is used to predict the parameters of an affine (or rigid) transformation model, with 12 (or 6) degrees of freedom.
+
+## Model training
+Registration networks are trained using standard minibatch gradient descent, with each minibatch sampling pairs of moving-fixed images and, when available, moving-and-fixed labels.
+
+`DeepReg`, at its first release, provides six predefined dataset loaders for three types of image pairs, unpaired, grouped and paired, each with two dataset loaders depends on whether corresponding labels are avaialble. Random sampling methods are used for unpaired images. For grouped images, group will be sampled before randomly sampling intra-group images for train without biasing to different groups. When multiple labels are avaialble associated with each image, another staged sampling methods are used to draw one pair of labels for each sampled image pair, during each training iteration. Details of the available dataset loaders, how to add cutomised loader and the sampling options are provided in a series of tutorials under the `tutorials` folder.
+
+## The software
+`DeepReg` is unit tested, and includes both technical documentation and various tutorials to introduce users to various use cases using open-sourced datasets. The continuous integration platform TravisCI (https://travis-ci.org/) is used together with a cutomised contributor guideline to facilitate contributions.
+
+
+# Example applications
+`DeepReg` provides a collection of demonstrations, _DeepReg Demos_, using open-accesible data with real clinical applications.
+
+## Paired images
+If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical application for tracking organ motion and other temporal changes between two time points require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expritory and inspritory, is such an example of both unsupervised (without labels) and combined supervision (with segmentation of anatomical structures). Furthermore, registering prostate MR, aquired before surgery, and the intra-operative trasnrectal ultrasound images is an example of weakly-supervised training for multimodal image registration.
+
+## Unpaired images
+Unpaired images are found in applications such as single-modality inter-subject registration. Two dataset loaders for labelled and unlabelled image data. Registering different MR images of brain from different subjects has played a fundamental role in population studies in neuroimaging. An example registering 3D ultrasound images from different surgical patients are also provided, especially to demonstrate the support for experiments such as cross-validation in DeepReg.
```

Comment:
- "Two dataset loaders for labelled and unlabelled image data." this sentence is incomplete
- Last sentence, An example registering ... "are" also provided, should be "is"